### PR TITLE
Update gems lock file 8.6

### PR DESCRIPTION
**Pull request by andsel**
Generated locally with
```sh
ruby tools/release/bump_plugin_versions.rb 8.6 8.5.1 minor --no-pr
```

### CODE REVIEW

File: Gemfile.jruby-2.6.lock.release [hunk 1]
```diff
@@ -903,8 +896,15 @@ DEPENDENCIES
   logstash-integration-jdbc
   logstash-integration-kafka
   logstash-integration-rabbitmq
+  logstash-mixin-ca_trusted_fingerprint_support
```
Comment (mashhurs) [inline Gemfile.jruby-2.6.lock.release:899]: These lines are interesting to me. AFAIK mixin is already included and now sure why they are not included in prev versions?

Comment (andsel) [reply Gemfile.jruby-2.6.lock.release:899]: Neither of this mixin were listed, I think was due to the old requirement:
```
logstash-output-elasticsearch (>= 11.6.0)
```

**Review by mashhurs — APPROVED** [review]
LGTM!

**Review by jsvd — CHANGES_REQUESTED** [review]
Generating from the [github action](https://github.com/elastic/logstash/actions/runs/3515418082) makes for a slightly cleaner PR. This PR removes the es output constraint existing in Gemfile.template from the lockfile, which shouldn't.
I suggest we merge https://github.com/elastic/logstash/pull/14769/files instead

**Comment by andsel** [comment]
Superseeded by #14769

**Pull request closed without merging**